app/src/main/res/values/strings.xml

```diff
@@ -217,4 +217,8 @@
     <string name="continue_with_empty_fields">Continue with empty fields?</string>
     <string name="continue_action">Continue</string>
     <string name="chose_image_source">Choose image source</string>
+
+    <!--    Notifications-->
+    <string name="channel_name">Channel name</string>
```

Comment:
Could it be  :
-  O-Fish notifications
-  O-Fish will show on duty reminder each 10 hours?

Reply:
Done